internal/twirptest/proto/proto_test.go

```diff
@@ -13,8 +13,129 @@
 
 package proto
 
-import "testing"
+import (
+	context "context"
+	fmt "fmt"
+	ioutil "io/ioutil"
+	http "net/http"
+	"net/http/httptest"
+	strings "strings"
+	"testing"
+	"time"
+
+	twirp "github.com/twitchtv/twirp"
+)
 
 func TestCompilation(t *testing.T) {
 	// Test passes if this package compiles
 }
+
+type errReader int
```

Comment:
Adding the test in `/proto/proto_test.go`,  feels a little arbitrary, because this is not related to testing an empty proto message. What about moving the test to `service_test.go`? all other arbitrary tests are there